src/commands/android/subcommands/install/avd.ts

```diff
@@ -0,0 +1,129 @@
+import colors from 'ansi-colors';
+import inquirer from 'inquirer';
+
+import Logger from '../../../../logger';
+import {Platform} from '../../interfaces';
+import {getBinaryLocation} from '../../utils/common';
+import {execBinaryAsync, execBinarySync} from '../../utils/sdk';
+import {getInstalledSystemImages, showMissingBinaryHelp} from '../common';
+
+type DeviceType = 'Nexus' | 'Pixel' | 'Wear OS' | 'Android TV' | 'Desktop' | 'Others';
+
+const deviceTypesToGrepCommand: Record<DeviceType, string> = {
+  'Nexus': 'Nexus',
+  'Pixel': 'pixel',
+  'Wear OS': 'wear',
+  'Android TV': 'tv',
+  'Desktop': 'desktop',
+  'Others': '-Ev "wear|Nexus|pixel|tv|desktop"'
+};
+
+export async function createAvd(sdkRoot: string, platform: Platform): Promise<boolean> {
+  try {
+    const avdmanagerLocation = getBinaryLocation(sdkRoot, platform, 'avdmanager', true);
+    if (!avdmanagerLocation) {
+      showMissingBinaryHelp('avdmanager');
+
+      return false;
+    }
+
+    const sdkmanagerLocation = getBinaryLocation(sdkRoot, platform, 'sdkmanager', true);
+    if (!sdkmanagerLocation) {
+      showMissingBinaryHelp('sdkmanager');
+
+      return false;
+    }
+
+    const avdNameAnswer = await inquirer.prompt({
+      type: 'input',
+      name: 'avdName',
+      message: 'Enter a name for the AVD:'
+    });
+    const avdName = avdNameAnswer.avdName;
+
+    const installedSystemImages: string[] = await getInstalledSystemImages(sdkmanagerLocation, platform);
+    if (!installedSystemImages.length) {
+      return false;
```

Comment:
Error out here that no system image is installed, and then provide a command that user can use to install a new system image.